packages/docs/docs/getting-started/hierarchy.mdx

```diff
@@ -16,6 +16,14 @@ the Document Object Model used ot represent HTML and XML documents.
 Here's an example of a simple scene hierarchy together with its object
 representation:
 
+:::info
+
+If you are using Firefox make sure to check if `dom.textMetrics.fontBoundingBox`
```

Comment:
I think we should turn this into a component:
```tsx
import Admonition from '@site/src/theme/Admonition';
import Link from '@docusaurus/Link';

export default function FirefoxTextWarning() {
  return (
    <Admonition type="caution">
      If you are using Firefox make sure to check if
      `dom.textMetrics.fontBoundingBox` is enabled else you might not see the{' '}
      <Link to="/api/2d/components/Text">
        <code>Text</code>
      </Link>
      . You can enable this by navigating to <code>about:config</code>.
    </Admonition>
  );
}
```

And then use it in markdown:
```tsx
import FirefoxTextWarning from '@site/src/warnings/FirefoxTextWarning';
// ...
<FirefoxTextWarning />
```

This will make it easier to find and remove all these notices in the future, once the bug is fixed.

Reply:
Yeah, totally agree. Done it this way now.